nss-day-cohort-52/giffygram-retro-blasters
# Added navBar/ userProfile/ fixed CSS

#### Changes Made
1. Added file `UserProfile` to `feed` directory.
2. Added file `navBar` to `nav` directory.
1. Modified file `GiffyGram` to include `navBar/userProfile event listeners` and functionality.
2. Modified file `PostList` to include `user link` and functionality.
3. Modified file `provider` to include `getUserProfile setUserProfile` and functionality.
4. Modified file `css` and functionality.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout branchname
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > When you click the name of the user who posted, it takes you to the user profile
    > The navBar stays at the top


4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
